crates/ruff/src/checkers/ast/analyze/bindings.rs

```diff
@@ -0,0 +1,71 @@
+use crate::checkers::ast::Checker;
+use crate::codes::Rule;
+use crate::rules::{
+    flake8_import_conventions, flake8_pyi, pyflakes, pylint,
+};
+use ruff_diagnostics::{Diagnostic, Fix};
+
+/// Run lint rules over the [`Binding`]s.
+pub(crate) fn bindings(checker: &mut Checker) {
```

Comment:
This is `Checker::check_bindings`, but moved out of the file (and out of the struct). Arguably it should be an associative function, but it felt weird to put multiple `impl` blocks for `Checker` in so many different files. Anyway, IMO this is fine.